src/sas/qtgui/Plotting/Plotter.py

```diff
@@ -222,6 +222,10 @@ def plot(self, data=None, color=None, marker=None, hide_error=False, transform=T
                             zorder=1,
                             picker=True)
 
+        # Display horizontal axis if requested
+        if data.show_yzero:
+            ax.axhline(color='black', linewidth=1)
```

Comment:
I find `linewidth=1` slightly thick - the same thickness as axes. Maybe 0.5 would look nicer? or not...

Reply:
I wonder if there's a way to automatically set it to the thickness of the borders.

Reply:
> I wonder if there's a way to automatically set it to the thickness of the borders.

Maybe something like `ax.axhline(color='black', linewidth=self.ax.spines['left'].get_linewidth())`?